containers/Containerfile

```diff
@@ -62,17 +53,13 @@ RUN yum install --disablerepo=updates -y \
     python3-rpm \
     python3-simplejson \
     python3-six \
-    python3-sphinx \
     python3-virtualenv \
-    rarian-compat \
     redhat-rpm-config \
     rpm-build \
     rpmlint \
     subscription-manager-rhsm-certificates \
     sudo \
-    vim \
     wget \
-    zsh \
```

Comment:
vim and zsh will allow for local development on people's development machines since toolbox tries to transparently insert a developer into a sandbox.  Part of the benefit of doing this would be to spin up one of these PRs locally and debug using the developer's environment.   (I'm not sure who on your team uses zsh but @cnsnyder  definitely uses vim)

Reply:
I can add back `vim`, but I don't think that anyone uses `zsh`.

Reply:
I think I'm the only zsh person haha